docs/notes/todo.md

```diff
@@ -0,0 +1,45 @@
+# Todo
+
+
+## Critical Path Tasks
+
+- [ ] Test and debug Registry routines
+- [ ] Implement file cache
+- [ ] Implement paging
+
+
+## These Tasks Would Be Nice
+
+- [ ] Separate HAL into KM DLL
+
+
+## Function Groups Totally or Partially Unimplemented
+
+- [ ] Dma functions (see hal/x86/dma.c)
+- [ ] Shutdown support (see ex/power.c)
+- [ ] Zw(Set/Get)SystemInformation (see ex/sysinfo.c)
```

Comment:
This is implemented; there may be some information classes that are unimplemented, but those are very uncommonly used.